src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -1121,6 +1122,22 @@ void network::execute_impl(const std::vector<event::ptr>& events) {
                        << data_shape_str.str() << std::endl;
     }
 
+    GPU_DEBUG_IF(!debug_config->dump_graphs.empty() && debug_config->is_target_iteration(curr_iter)) {
+        auto get_fixed_str = [](int value, int length = 2) -> std::string {
+            std::ostringstream ss;
+            ss << std::setw(length) << std::setfill('0') << std::to_string(value);
+            return ss.str();
+        };
+        std::string path = get_dir_path(get_config());
+        if (!path.empty()) {
+            std::ofstream ofs(path + "cldnn_program_exec_" + get_fixed_str(get_id()) + "_" + get_fixed_str(get_program()->get_id())
```

Comment:
Something like `0_0_0` would be difficult to understand. Could you make it something like `p0_n0_0`?